bcgov/cas-registration
# Revise User Operator form

https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=50189801

To test the workflows in this PR:
- [x] user requests access to an existing operator that has an admin
-- Login as bc-cas-secondary
-- Delete the bc-cas-secondary record in the user operator table: `delete from erc.user_operator where id>2;`  (or temporarily edit the fixtures)
-- Request access to Existing Operator 2
-- You should see the correct confirmation message and no form
- [x] user requests access to an existing operator that DOESN'T have admin
-- login with any bceid and clear the user_operator table (`truncate erc.user_operator;` or adjust the fixtures)
-- request access to any operator
-- You should see the correct confirmation message and no form
- [x] user adds new operation
 -- login with any bceid and clear the user_operator table (`truncate erc.user_operator;` or adjust the fixtures)
-- add an operator
-- fill out the first page of the form and submit (there should no longer be a second page)
-- You should see the correct confirmation message on submission
- [x] edit an operator
--reset the db and login with any bceid
--click the my operator tile and you should be directed to the operator form
-- there should be no second page
-- editing should work without errors and you should be redirected to the dashboard

Since I touched the MultiStepForm component, which is used in every form, I manually checked that still works and it does other than this pre-existing bug: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=52418646

If you back into the operator tile after requesting access, you should see the same confirmation message you saw when requesting access the first time. For me, the screen flashes here, which seems to be a known next bug: https://github.com/vercel/next.js/issues/57257
